app/containers/RoomHeader/RoomHeader.tsx

```diff
@@ -160,7 +160,7 @@ const Header = React.memo(
 		let scale = 1;
 		const isMasterDetail = useAppSelector(state => state.app.isMasterDetail);
 		const subtitleAccessibilityLabel = tmid ? parentTitle : subtitle;
-		const accessibilityLabel = `${statusAccessibilityLabel} ${title} ${subtitleAccessibilityLabel}.`;
+		const accessibilityLabel = `${statusAccessibilityLabel} ${title} ${subtitleAccessibilityLabel ?? ''}.`;
```

Comment:
Similar issue.
If `subtitleAccessibilityLabel === ''`, `??` isn't going to use the second argument either way.
You may want to use `||` instead.